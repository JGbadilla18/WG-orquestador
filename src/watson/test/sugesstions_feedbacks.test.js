const testWatson = require('./testWatson');

watsonText = 'Donde me puedo quejar..'

it(watsonText, () => {
  expect.assertions(1);
  return testWatson(watsonText).then(data => expect(data).toMatchSnapshot());
});
