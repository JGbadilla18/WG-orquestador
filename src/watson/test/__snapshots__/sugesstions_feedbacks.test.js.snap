// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Donde me puedo quejar.. 1`] = `
Object {
  "actions": Array [],
  "components": Array [
    Object {
      "response_type": "text",
      "text": "Puedes comunicarte con nosotros llamando al número 2411-6010 en donde agentes especializados con gusto te escucharán y resolverán todos tus comentarios.",
    },
    Object {
      "response_type": "pause",
      "time": 10000,
      "typing": true,
    },
  ],
}
`;
